kubernetes-elasticsearch-cluster/es-data-stateful.yaml

```diff
@@ -13,13 +13,15 @@ spec:
       component: elasticsearch
       role: data
   serviceName: elasticsearch-data
-  replicas: 3
+  replicas: 1
```

Comment:
[Kubenetes docs say](https://kubernetes.io/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/): "A Kubernetes cluster that handles production traffic should have a minimum of three nodes." So this should be 3. Same for rest of PR.